files/ru/web/api/html_drag_and_drop_api/drag_operations/index.md

```diff
@@ -231,24 +231,24 @@ However, you can also update the user interface with an insertion point or highl
 }
 ```
 
-In this example, the element with the class `droparea` will receive a 1 pixel black outline while it is a valid drop target, that is, if the {{domxref("Event.preventDefault","preventDefault()")}} method was called during the `{{event("dragenter")}}` event.
+In this example, the element with the class `droparea` will receive a 1 pixel black outline while it is a valid drop target, that is, if the {{domxref("Event.preventDefault","preventDefault()")}} method was called during the `[`dragenter`](/ru/docs/Web/Events/dragenter)` event.
 
 > [!NOTE]
-> You must cancel the `{{event("dragenter")}}` event for this pseudoclass to apply, as this state is not checked for the `{{event("dragover")}}` event.
+> You must cancel the `[`dragenter`](/ru/docs/Web/Events/dragenter)` event for this pseudoclass to apply, as this state is not checked for the `[`dragover`](/ru/docs/Web/Events/dragover)` event.
 
-For more complex visual effects, you can also perform other operations during the `{{event("dragenter")}}` event. For example, by inserting an element at the location where the drop will occur. This might be an insertion marker, or an element that represents the dragged element in its new location. To do this, you could create an [image](/ru/docs/XUL/image) or [separator](/ru/docs/XUL/separator) element and simply insert it into the document during the `{{event("dragenter")}}` event.
+For more complex visual effects, you can also perform other operations during the `[`dragenter`](/ru/docs/Web/Events/dragenter)` event. For example, by inserting an element at the location where the drop will occur. This might be an insertion marker, or an element that represents the dragged element in its new location. To do this, you could create an [image](/ru/docs/XUL/image) or [separator](/ru/docs/XUL/separator) element and simply insert it into the document during the `[`dragenter`](/ru/docs/Web/Events/dragenter)` event.
```

Comment:
```suggestion
For more complex visual effects, you can also perform other operations during the [`dragenter`](/ru/docs/Web/Events/dragenter) event. For example, by inserting an element at the location where the drop will occur. This might be an insertion marker, or an element that represents the dragged element in its new location. To do this, you could create an [image](/ru/docs/XUL/image) or [separator](/ru/docs/XUL/separator) element and simply insert it into the document during the [`dragenter`](/ru/docs/Web/Events/dragenter) event.
```